### DISCUSSION TITLE
Can´t access weights through git hub

### DISCUSSION THREAD

**Discussion by lucasthedevoman** [community/community#147116] [Other Feature Feedback, Questions, & Ideas]
### Select Topic Area

Question

### Body

I accidently deleted some cookies and to one´s utter discontent discovered that weights was no longer playcating gift hub, to complicate matters I can´t bypass the phone verification on "paperspace" HELP!!

**Comment by lucasthedevoman** [comment]
Should such a thing not pan out could someone offer me contingencies on how to traverse such a dillema

**Comment by mecodeatlas** [comment]
Hi @lucasthedevoman,

Thanks for being a part of the GitHub Community, we're glad you're here!

If you're looking for help for this specific topic, you might want to try asking for help somewhere that focuses on this project, such as the [Paperspace FAQ](https://www.paperspace.com/faq-cloud-gpu). It's possible that another GitHub user might have run into this same issue and can help, but the GitHub Community Discussions focuses primarily on topics related to GitHub itself or collaboration on project development and ideas. We want to make sure you’re getting the best support you can, but this space may not be the right place for this particular topic.

Best of luck!
